docs/eventing/samples/iot-core/README.md

```diff
@@ -201,3 +201,35 @@ see them in the subscriber.
         -src "iot-core demo" \
         -events 10
     ```
+
+### Cleanup
+
+To cleanup the knative resources:
+
+1.  Remove the gcppubsub controller:
+    ```shell
+    pushd $HOME/go/src/github.com/knative/eventing-sources
```

Comment:
Instead of using `pushd`, how about use `cd` instead?

Reply:
We can do that, but I wanted to be consistent with the setup instructions.  I think it'd make sense to use the same (whichever that is) in both places.  What would you prefer?

Reply:
+1 to keep consistent, but I'd prefer that we use `cd` instead. I think we can handle this in other PR.